runtimes/python/python_apps.md

```diff
@@ -53,23 +53,19 @@ Remove the environment variable or set it to `false` to disable this feature.
 If you have any question, feel free to [contact our support](https://www.clever-cloud.com/doc/get-help/support/).
 
 
-## Configuration file
+## Configuring the deployment
 
-The configuration file for your Python application must be `/clevercloud/python.json`.
-
-You can find a [example of Flask application](/doc/python/python-flask-sample-app/) who use concept detailed below.
 
+To make the deployment process aware of what you are trying to run, you will use some
+environment variables. To set such environment variables, go in the "Environment
+variables" menu of your application.
```

Comment:
Python apps can be launched in a variety of ways. You can specify how to start your application (for instance which module to run) by setting environment variables (in the "Environment variables" section of the console).

Something like that would be a bit better IMO